c_glib/arrow-glib/compute.cpp

```diff
@@ -3103,6 +3112,15 @@ garrow_sort_key_class_init(GArrowSortKeyClass *klass)
                            static_cast<GParamFlags>(G_PARAM_READWRITE |
                                                     G_PARAM_CONSTRUCT_ONLY));
   g_object_class_install_property(gobject_class, PROP_SORT_KEY_ORDER, spec);
+
+  spec = g_param_spec_enum("null_placement",
```

Comment:
```suggestion
  /**
   * GArrowSortKey::null-placement:
   *
   * Whether nulls and NaNs are placed at the start or at the end.
   *
   * Since: 15.0.0
   */
  spec = g_param_spec_enum("null_placement",
```

Reply:
done